### PR TITLE
chore: update script to include strava challenge

### PR DESCRIPTION
**Pull request by ottbunn**
# Summary
update script to include strava challenge

### CODE REVIEW

**Review by ryanycw — APPROVED** [review]
Perfect!!

File: contracts/script/D01DeployTracker.s.sol [hunk 1]
```diff
@@ -35,9 +35,18 @@ contract Deploy is Script {
...
-            address(0xc108B7256052137a1b785192E172670f3BC8dCD6),
+            address(0x29C3d6b54E2F8Ae641Fc331cF2143B6d05c97897),
             0.003 ether
         );
+        tracker.register(
+            address(0x29C3d6b54E2F8Ae641Fc331cF2143B6d05c97897),
+            "Run 30 mins 10 times",
+            5,
+            block.timestamp + 10,
+            block.timestamp + 1000,
```
Comment (antoncoding) [inline contracts/script/D01DeployTracker.s.sol:46]: can we use a much better timestamp for end timestamp?

**Review by antoncoding — APPROVED** [review]

File: contracts/script/D01DeployTracker.s.sol (hunk 1 above)
Comment (ryanycw) [reply contracts/script/D01DeployTracker.s.sol:46]: By better do you mean longer lol?

Comment (antoncoding) [reply contracts/script/D01DeployTracker.s.sol:46]: yes xD

**Pull request merged**